src/api/spec/controllers/attribute_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe AttributeController, type: :controller do
+  render_views
+
+  describe '#update' do
+    let!(:admin) { create(:admin_user) }
+    let(:user1) { create(:confirmed_user) }
+    let!(:namespace) { create(:attrib_namespace, name: 'TEST') }
+    let(:attribute) { create(:attrib_type, attrib_namespace: namespace) }
+    let(:dummy) { 'Dummy' }
+
+    let(:data1) do
```

Comment:
:thought_balloon: why not use a factory and `.to_xml` on it?

Reply:
Hmm, I would I have to_xml without creating it?

Reply:
`build(:my_factory).to_xml` :wink:

Reply:
And please using speaking names. What kind of data is this?

Reply:
there is no to_xml on attrib_types - so now?